main.go

```diff
@@ -97,9 +103,16 @@ func parseArgsAndLoadConf() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	if conf.Host == "" {
+		conf.Host = host
+	}
+	if conf.Port == 0 {
+		conf.Port = port
+	}
```

Comment:
Could we just use the config file for this? I'm not sure if it's important to support both (and I had to read the code twice to make sure I understand what was happening here.)

Reply:
Done, removed the flag